packages/node_modules/@ciscospark/widget-recents/src/components/space-item/index.js

```diff
@@ -25,7 +25,11 @@ export default function SpaceItem({
   type
 }) {
 
-  function handleClick() {
+  function handleClick(e) {
```

Comment:
Is this supposed to be in the "handle click" function? What happens if you actually click?

Reply:
Yea, i cheated a bit here... probably should have a separate method for handle keypress

Reply:
Yea gotcha. I usually do too